### PR TITLE
Improvment/has name

### PR DESCRIPTION
**Pull request by Redaloukil**

### CODE REVIEW

File: src/utils/hasName.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const _hasName = baladyiaName => {
```
Comment (Fcmam5) [inline src/utils/hasName.js]: Please rename it to `hasName` here and in the places where you import it. Also rename `baladyiaName` to something like `givenName`

File: src/index.js [hunk 2]
```diff
@@ -39,6 +40,7 @@ module.exports = {
     wilayaProjection,
     isValidWilayaCode,
     isValidZipCode,
+    hasName,
```
Comment (Fcmam5) [inline src/index.js]: I think that it's okay to not export this function as we're only using it internally. If you see that it make sense to do it, we'd require to add a jsDoc to the function, and we'll need to update the README

Comment (Redaloukil) [reply src/index.js]: For now I see that it doesn't indeed , maybe for next usecases it can be usable but not for not as i know.

Comment (Fcmam5) [reply src/index.js]: So, let's not export it please

**Review by Fcmam5 — APPROVED** [review]
Thank you Reda!

**Pull request merged**